hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java

```diff
@@ -151,6 +155,7 @@ public String toString() {
       + ", help=" + help
       + ", supportTimestamp=" + supportTimestamp
       + ", decodePartition=" + decodePartition
+      + ", createManagedTable= " + createManagedTable
```

Comment:
`+ ", createManagedTable=" + createManagedTable` to align with others?

Reply:
done!